framework-docs/modules/ROOT/pages/index.adoc

```diff
@@ -1,24 +1,24 @@
 :noheader:
+[[spring-framework-documentation]]
 = Spring Framework Documentation
-include::attributes.adoc[]
 
 [horizontal]
-<<overview.adoc#overview, Overview>> :: History, Design Philosophy, Feedback,
+xref:overview.adoc[Overview] :: History, Design Philosophy, Feedback,
 Getting Started.
-<<core.adoc#spring-core, Core>> :: IoC Container, Events, Resources, i18n,
+xref:core.adoc[Core] :: IoC Container, Events, Resources, i18n,
 Validation, Data Binding, Type Conversion, SpEL, AOP, AOT.
 <<testing.adoc#testing, Testing>> :: Mock Objects, TestContext Framework,
 Spring MVC Test, WebTestClient.
-<<data-access.adoc#spring-data-tier, Data Access>> :: Transactions, DAO Support,
+xref:data-access.adoc[Data Access] :: Transactions, DAO Support,
 JDBC, R2DBC, O/R Mapping, XML Marshalling.
-<<web.adoc#spring-web, Web Servlet>> :: Spring MVC, WebSocket, SockJS,
+xref:web.adoc[Web Servlet] :: Spring MVC, WebSocket, SockJS,
 STOMP Messaging.
-<<web-reactive.adoc#spring-web-reactive, Web Reactive>> :: Spring WebFlux, WebClient,
+xref:testing/unit.adoc#mock-objects-web-reactive[Web Reactive] :: Spring WebFlux, WebClient,
```

Comment:
that's the wrong link

```suggestion
xref:web-reactive.adoc[Web Reactive] :: Spring WebFlux, WebClient,
```

Reply:
Thank you for finding this. I've fixed it via 5e43a7a7ca36ff63d744a35b156e816370d12284